src/App.js

```diff
@@ -1,25 +1,14 @@
-import logo from './logo.svg';
 import './App.css';
+import Display from './Display';
+import ButtonsPanel from './ButtonPanel';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
-}
+const App = () => (
+  <div className="App">
+    <header className="App-header">
+      <Display />
+      <ButtonsPanel />
+    </header>
```

Comment:
- [ ] It will be great to wrap the children of this component in ` fragment`. This ensures that there are no unnecessary dom elements created in your application. To learn more about this kindly read this [article](https://reactjs.org/docs/fragments.html#short-syntax). ⬇️ 
![ReactFragment](https://user-images.githubusercontent.com/32361963/128009198-0da749c4-ebc9-4e06-ae1f-f89f10da05b2.png)